site/_layouts/doc.liquid

```diff
@@ -118,7 +118,7 @@
                     <li {% if page.url == "/docs/v1/tech/migration-guide/tutorial.html" %}class="active"{% endif %}><a href="/docs/v1/tech/migration-guide/tutorial">Tutorial</a></li>
                   </ul>
                 </li>
-                <li {% if page.navcategory == "admin" or page.url contains "/admin-guide/" %} class="open"{% endif %}>
+                <li {% if page.navcategory == "admin" or page.url contains "/admin-guide/" or page.url contains "/reference/" %} class="open"{% endif %}>
```

Comment:
I noticed that selecting one of the pages under `Admin Guides > Reference` was not expanding any of the sections.